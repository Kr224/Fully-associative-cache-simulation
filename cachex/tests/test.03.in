1024
65536
4
22 
1534
22 
1534
stats
