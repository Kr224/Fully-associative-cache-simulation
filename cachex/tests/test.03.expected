Loaded value [0xb9cb17b29e5109d2] @ address 0x00000016
Loaded value [0x21e774785a710d44] @ address 0x000005fe
Loaded value [0xb9cb17b29e5109d2] @ address 0x00000016
Loaded value [0x21e774785a710d44] @ address 0x000005fe
Cache hits: 2, misses: 2 -- hit rate 50%
